pkgs/development/libraries/hunspell/dictionaries-chromium.nix

```diff
@@ -90,4 +90,20 @@ rec {
     dictFileName = "fr-FR-3-0.bdic";
     shortDescription = "French (France)";
   };
+
+  # PORTUGUESE
+
+  pt_PT = pt-pt;
+  pt-pt = mkDictFromChromium {
+    shortName = "pt-pt";
+    dictFileName = "pt-PT-3-0.bdic";
+    shortDescription = "Portuguese (Portugal)";
+  };
+
+  pt_BR = pt-br;
```

Comment:
I have no idea what this line does, but it is in all other dicts so i kept the structure.

If reviewer thinks this is useless, i will open PR removing all similar lines.